common/algorithms/simd-int-to-string-test.cpp

```diff
@@ -11,23 +11,27 @@
 
 namespace {
 std::string int32_to_string(int32_t x) {
-  char buffer[11];
-  return std::string{buffer, simd_int32_to_string(x, buffer)};
+  const size_t buffer_size = 11;
+  char buffer[buffer_size];
+  return std::string{buffer, simd_int32_to_string(x, buffer, buffer_size)};
```

Comment:
please, just `return {buffer, simd_int32_to_string(x, buffer, buffer_size)};`,
or given in mind previous comment `return {buffer, simd_int32_to_string(x, buffer, buffer.size())};`

Reply:
same for code below

Reply:
done